src/pages/settings/Security/CloseAccountPage.js

```diff
@@ -30,6 +30,9 @@ const propTypes = {
         /** Error message if previous attempt to close account was unsuccessful */
         error: PropTypes.string,
 
+        /** Error message keyed by microtime if previous attempt to close account was unsuccessful */
+        errors: PropTypes.object,
```

Comment:
Update it to an object of strings to fix lint error.
https://github.com/Expensify/App/blob/4dc7fef0ec778be71114898bdf7d56ed363bee0a/src/pages/workspace/withPolicy.js#L45-L46